src/foam/nanos/crunch/MinMaxCapability.js

```diff
@@ -67,6 +67,11 @@ foam.CLASS({
       },
       includeInDigest: false,
     },
+    {
+      class: 'String',
+      name: 'customCardSelectView',
+      value: 'foam.u2.view.CardSelectView'
+    }
```

Comment:
This should be handled by faceting, this property should not be required

Reply:
Sarthak is correct: https://github.com/kgrgreer/foam3/blob/development/src/foam/u2/view/CardSelectView.js#L17

There must be a problem with the 'of' not being passed wherever CardSelectView is called